WhisperAPI/WhisperAPI/Models/SearchQuerry.cs

```diff
@@ -13,6 +13,9 @@ public enum MessageType
             Agent = 1
         }
 
+        [Key]
+        public int Id { get; set; }
+
```

Comment:
You could add public bool Relevant {get; set;} to sstore the relevance of a query. Make sure that this field is not required when you send the json from salesforce to whisperapi.